ClosedXML/Excel/CalcEngine/Functions/Text.cs

```diff
@@ -275,6 +276,63 @@ private static object _Text(List<Expression> p)
             else
                 return nf.Format(number, CultureInfo.InvariantCulture);
         }
+        /// <summary>
+        /// A function to Join text https://support.office.com/en-us/article/textjoin-function-357b449a-ec91-49d0-80c3-0e8fc845691c
+        /// </summary>
+        /// <param name="p">Parameters</param>
+        /// <returns> string </returns>
+        /// <exception cref="ApplicationException">
+        /// Delimiter in first param must be a string
+        /// or
+        /// Second param must be a boolen (TRUE/FALSE)
+        /// </exception>
+        private static object TextJoin(List<Expression> p)
+        {
+            if( p.Count > 254) throw new ApplicationException("Max number of params 254");
```

Comment:
I suspect this check is not necessary since the maximum number of parameters is set in `RegisterFunction`. But to be completely sure I'd add a test for this too...

Reply:
You are correct this test is not necessary. Do you still want a test case for it?